Source/RapyutaSimulationPlugins/Public/Core/RRConversionUtils.h

```diff
@@ -152,15 +152,29 @@ class URRConversionUtils : public UBlueprintFunctionLibrary
         OutputZ = InputZ * 100.f;
     }
 
+    /**
+     * @brief Convert ROS Rotation Euler (rad) to UE Vector retaining unit as [rad]
+     * @param Input
+     * @return FVector
+     */
     UFUNCTION(BlueprintCallable, Category = "Conversion")
-    static FVector RotationROSToUE(const FVector& Input)
+    static FVector RotationROSToUEVector(const FVector& Input)
```

Comment:
Based on the naming convention, i.e. ROSToUE does orieantation and unit conversion, it is better to rad to degree here

Reply:
So we actually use it also for Odom msg conversion [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Public/Core/RRConversionUtils.h#L117), where no rad<->deg should happen.
I added a param for conversion up to the caller.

Reply:
not 100% sure but it seems bug and OdomROSToUE should return rad since OdomData in ROS is radian.